cmd/internal/docker/docker.go

```diff
@@ -213,20 +211,20 @@ func (c *Container) Wait(ctx context.Context, timeout time.Duration) error {
 		timeout = time.Minute
 	}
 	var (
-		done     = time.After(timeout)
-		drv      = c.Driver()
-		dsn, err = c.DSN()
+		done   = time.After(timeout)
+		u, err = c.URL()
 	)
 	if err != nil {
 		return err
 	}
 	for {
 		select {
 		case <-time.After(100 * time.Millisecond):
-			db, err := sql.Open(drv, dsn)
+			client, err := sqlclient.Open(ctx, u)
 			if err != nil {
-				return err
+				continue
 			}
+			db := client.DB
```

Comment:
Why did you change this from the `database/sql` to `sqlclient`? It is only meant to check for availablitlity. Also, you need to close the `sqlclient`, correct?

Reply:
Because I deleted the DSN function, as it's implemented by the drivers.

I'm closing the DB, but will switch to closing the client.

Reply:
Rethinking it. No need to close the client, closing the underlying DB is fine.